ldms/src/ldmsd/ldmsd_failover.c

```diff
@@ -2079,7 +2086,7 @@ int failover_cfgprdcr_handler(ldmsd_req_ctxt_t req)
 	}
 
 	p = ldmsd_prdcr_new_with_auth(name, xprt, host, atoi(port), ptype,
-			atoi(interval), auth, _uid, _gid, _perm);
+			atoi(interval), auth, _uid, _gid, _perm, atoi(cache_ip));
```

Comment:
@nichamon, did we check what would happen if we run a 4.4.5 ldmsd_controller/maestro against a 4.4.3 ldmsd? As long as it ignores the extra parameter I think we're fine, but it shouldn't crash.